ci/awards.py

```diff
@@ -0,0 +1,17 @@
+import json
+import yaml
+import requests
+import os
+
+r = requests.get(
+  'https://www.thebluealliance.com/api/v3/team/frc2521/awards',
+  headers={'X-TBA-Auth-Key': os.getenv('tba_api_key')}
+)
+
+awards = [{
+  'name': a['name'].split(' sponsored by')[0],
+  'event': a['event_key'],
```

Comment:
Eventually should we fill this in with the name of the competition itself rather than the event key?

I'm thinking we could make a request an API request to `/event/{event_key}/simple` and get the value for the  `name` key. That's a lot of requests though so I doubt it's the best way to do it. Do you have any suggestions?

Reply:
I can also merge it in as-is right now and we can worry about it later

Reply:
This is tough without making lots more API calls. It's doable, but for the moment I was thinking of just listing the award and linking to the event's award page.